app/views/groups/show.html.erb

```diff
@@ -1,30 +1,29 @@
 <div class="container">
-  <div class="rounded bg-primary p-5 mt-3">
+  <div class="rounded bg-primary p-4 mt-3">
     <h2><%= @group.name %></h2>
     <p><%= @group.description %></p>
 
-    <p class="post-count"><%= @group.posts.count > 0 ? pluralize(@group.posts.count, 'post') : "No Post" %></p>
+    <p class="post-count"><%= @group.posts.count > 0 ? pluralize(@group.posts.count, 'posts') : "No Posts yet" %></p>
```

Comment:
this should be singular I think https://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize

Reply:
this whole line should be ```pluralize(@group.posts.count, 'post')```